blog/content/introtogonum.md

```diff
@@ -0,0 +1,155 @@
+ease-of-use.
+Gonum is different from other popular scientific libraries, in that the Go language
+does not allow operator or method overloading.
+On the surface, this often means that the same algorithm requires a few more
+lines of code in Go.
+We fully recognize that sexy one-liners are rare with Gonum.
+
+We accept this cost because we believe it comes with an enormous benefit.
+The Gonum function and method signatures make the algorithmic outputs clear,
+and it is easy to identify the code being executed and examine the implementation.
+Science is built on understanding how an experiment was performed and
+what the results are.
+We believe scientific code should be no different.
+In fact, Gonum is partly distinguished from other libraries in that the entire
+codebase can be run with pure Go -- you only need to know one programming
+language to read Gonum code (that said, it is also easy to call into existing C
+and Fortran libraries).
+This philosophy of simplicity has many follow-on benefits.
+For example, dense one-liners can be hard to understand, creating a barrier for 
+others to confirm correctness, and making future modifications difficult.
+Additionally, one-liners often obscure the necessary operations to execute that
+line of code.
+This creats a tension between code that looks pretty, and a targeted implementation
+that is efficient.
+In Gonum, there is usually only "one way to do things", creating a uniform style
+that in turn makes it easy to read algorithm implementations.
+
+The use of Go as a base language has many benefits beyond simplicity.
+Go is fast, approaching the speed of C.
+Go's language primitives are great for parallel computing, and in fact Gonum
+code can often be made to run in parallel by setting a flag.
+Go is designed for ["programming in the large"](https://talks.golang.org/2012/splash.article),
+a huge benefit for building an ecosystem of libraries that work with and build
+on top of one another.
+Go has extremely legible compile errors, full stack traces, trivial package downloading,
+automatic makefiles and import resolution, code formatting tools, easy code documentation,
+a race detector, and the list goes on.
+
+We believe in a new foundation for scientific computing, and we hope you'll join us.
+Please check back for more discussion on the philosophy of Gonum and the design
+decisions for specific packages.
+
+## Getting started with Gonum
+### Downloading the libraries
+First, you need to [install Go](https://golang.org/doc/install) if it is not
+already installed on your machine.
+It is recommended that you use the most recent version of Go.
+Run 
+```
+go get -u -t gonum.org/v1/gonum
+```
+to download the main set of Gonum packages.
+Then run
+```
+go get -u -t gonum.org/v1/plot
+```
+to download the plotting package.
+
+That's it!
+
+You can test that everything is working properly with 
+```
+cd ~/go/src/gonum.org/v1/gonum  (sub $GOPATH for ~/go if you have $GOPATH set)
```

Comment:
I would write `go test gonum.org/v1/...`. This obviates the need to even mention `GOPATH`.